packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -651,7 +651,8 @@ export function toTask(taskDto: TaskDto): theia.Task {
         result.execution = getProcessExecution(taskDto as ProcessTaskDto);
     }
 
-    if (taskType === 'shell') {
+    const execution = { command, args, options };
+    if (taskType === 'shell' || types.ShellExecution.is(execution)) {
```

Comment:
I wonder whether we need similar check for `taskType === 'process'` above?

Reply:
I think - no, the reasons I wrote [here](https://github.com/theia-ide/theia/pull/5591#discussion_r299494796). But let me know if I should to reconsider that.

Reply:
ok, please merge

Reply:
OK, thanks for testing changes and review!